src/BenchmarksServer/Startup.cs

```diff
@@ -319,10 +351,28 @@ private static string CloneRestoreAndBuild(string path, ServerJob job, string do
                 ["PATH"] = dotnetExeLocation + Path.PathSeparator + Environment.GetEnvironmentVariable("PATH")
             };
 
+            // Update the package and framework version in the props file
+
+            var dependenciesProps = Path.Combine(path, "benchmarks", "build", "dependencies.props");
```

Comment:
First, I think it's cleaner to modify MSBuild properties by passing them on the command line to `build.cmd|sh`, rather than modifying the `dependencies.props` file.  Second, I think you should modify the `Benchmarks*` properties that were explicitly added to make it easier to test other versions (https://github.com/aspnet/benchmarks/commit/8fa6759ea7ab145640a65b5739b8e6dd3df76bfd#diff-30dcc9789f8ea3c7632e4f054e60c072R12).  @dougbu Also has experience with this.

Reply:
Yes, this (as well as branches for testing Benchmarks against earlier runtimes) is why we use the separate projects in `Benchmarks.csproj`.